app/components/sanitize-html.js

```diff
@@ -1,8 +1,10 @@
 import Component from '@glimmer/component';
 import sanitizeHtml from 'sanitize-html';
+import merge from 'lodash/merge';
```

Comment:
Mocht er een betere manier zijn om dit type objecten te mergen, laat gerust iets weten bij review. Dit leek me de meest makkelijke manier zonder daar veel effort in te steken.

Reply:
@brenner-company je kan voor jouw use case hieronder volgens mij gewoon de [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) gebruiken: 

```
options.allowedAttributes = isPresent(options.allowedAttributes) ? options.allowedAttributes : { ...sanitizeHtml.defaults.allowedAttributes, ...additionalAllowedAttributes }; // add more attributes to the default allowed attributes
```

Dit heeft de voorkeur aangezien we dan geen extra library moeten importeren.
Beide objecten moeten wel bestaan natuurlijk.

Reply:
@tdn Als ik me herinner was het probleem hier dat ik geen shallow merge kon doen omdat de 'allowed attributes' ook arrays konden hebben en die dus ook correct moeten gemerged worden. Nu, kan desnoods wel eens zien of dit een vaste structuur is en ik desnoods een custom loop kan schrijven. Is dat een goed idee?

Reply:
aha, dan kan het inderdaad niet met de spread operator. Lodash is dan OK

Reply:
Nog even kort in detail bekeken en blijkbaar kan ook het volgende mee gegeven worden aan `sanitize-html`:

```
allowedAttributes: {
  iframe: [
    {
      name: 'sandbox',
      multiple: true,
      values: ['allow-popups', 'allow-same-origin', 'allow-scripts']
    }
  ]
}
```

De deep merge van Lodash zal dus wel effectief noodzakelijk zijn (ook in functie van eventuele latere aanpassingen binnen `sanitzie-html` zelf).